README.md

```diff
@@ -1,4 +1,6 @@
 # onboard-luiz-zanini
 
-*Project`s Name : Capivara-Branchs
-*Project`s Description : Utilizando git
+*Project`s Name : Capivara-Branchs  
+*Project`s Description : Utilizando git  
+*Environment and tools : MacBook PRO , node Version v10.16.0 , react-native 0.62, Language type-script, Xcode 11.2  
+*Steps to run and debug : on ./capivaraProjects/ios Using command ` npx react-native run-ios ` should start the application  
```

Comment:
No vscode tem como você ver como ficará o arquivo `.md` (Markdown).
<img width="1680" alt="Screen Shot 2020-05-04 at 18 34 48" src="https://user-images.githubusercontent.com/36424331/81016062-22b6b980-8e36-11ea-9a7d-6e6945765fc1.png">
<img width="1680" alt="Screen Shot 2020-05-04 at 18 35 35" src="https://user-images.githubusercontent.com/36424331/81016078-26e2d700-8e36-11ea-9170-1eab46d52b90.png">

Aqui acho que faltou um espaço entre o `*` e o segundo caracter. Acho que voce queria algo assim:
```
* Project`s Name : Capivara-Branchs  
* Project`s Description : Utilizando git  
* Environment and tools : MacBook PRO , node Version v10.16.0 , react-native 0.62, Language type-script, Xcode 11.2  
* Steps to run and debug : on ./capivaraProjects/ios Using command ` npx react-native run-ios ` should start the application  
```
* Project`s Name : Capivara-Branchs  
* Project`s Description : Utilizando git  
* Environment and tools : MacBook PRO , node Version v10.16.0 , react-native 0.62, Language type-script, Xcode 11.2  
* Steps to run and debug : on ./capivaraProjects/ios Using command ` npx react-native run-ios ` should start the application